### DISCUSSION TITLE
Secondary Nav Bar Question

### DISCUSSION THREAD

**Discussion by eightbit** [facebook/docusaurus#3885] [General]
Cool Project!!
[According to the docs](https://docusaurus.io/docs/en/navigation#secondary-on-page-navigation), the secondary nav bar is set with **onPageNav**.
I've installed v2 and I don't see that set anywhere in the code. The secondary Nav works, which is what I want, but in the interest of understanding how things work, has that been deprecated? If so, what controls the secondary nav?

Also, is it possible to add additional heading levels to the secondary nav?
Thanks.

**Comment by slorber** [comment]
Hi,

I think yes this is deprecated and v2 ships with a "table of content" on the top right part of docs.

It is provided through our MDX loader that parse markdown docs, find headings, and generate the TOC json structure, that is then rendered by a theme.

I don't think we have much customization options for now but let me know what you want to customize exactly
